### ISSUE TITLE
Save as broken

### ISSUE THREAD

**Issue by mbeckenbach** [martinroob/tiny-translator#51]
Looks like your last update broke something.

![image](https://user-images.githubusercontent.com/6654388/49295136-47766700-f4b5-11e8-8005-cf93ef9a2610.png)

**Comment by mbeckenbach** [comment]
We're using the same library:

```ts
import {saveAs} from 'file-saver';
...
 saveAs(f.blob, f.name);

```

**Comment by martinroob** [comment]
Sorry about that. Obviously my tests are not good enough.
Will fix it as soon as possible.

**Comment by mbeckenbach** [comment]
thx

**Comment by martinroob** [comment]
Fixed with v0.14.
In fact it was just the wrong import of file-saver.
